public/apps/configuration/panels/auth-view/auth-view.tsx

```diff
@@ -76,18 +98,26 @@ export function AuthView(props: AppDependencies) {
         <EuiTitle size="l">
           <h1>Authentication and authorization</h1>
         </EuiTitle>
-        {props.config.ui.backend_configurable && (
+        {!loading && !errorFlag && props.config.ui.backend_configurable && (
           <ExternalLinkButton
             href={DocLinks.BackendConfigurationDoc}
             text="Manage via config.yml"
           />
         )}
       </EuiPageHeader>
-      {/* @ts-ignore */}
-      <AuthenticationSequencePanel authc={authentication} loading={loading} />
-      <EuiSpacer size="m" />
-      {/* @ts-ignore */}
-      <AuthorizationPanel authz={authorization} loading={loading} config={props.config} />
+      {loading ? (
+        <EuiLoadingContent />
```

Comment:
Isn't this case already being handled by the AccessErrorComponent?

Reply:
for this specific component and how it is loaded, I had to handle it here. Also, these components do not render loading screens so the change between data-sources is not seamless